### DISCUSSION TITLE
Login error - docker version

### DISCUSSION THREAD

**Discussion by flastafari** [mitre/caldera#2933] [Q&A]
Hello, i'm having problem to login in Caldera.
When I put the login on platform and click in "Log In", nothing happens.
Could someone help me?

Tks in advance!

**Comment by un1c0rn-sec** [comment]
I'm having the same issue. It looks like the site's route is using `localhost` for the URL instead of the IP address of the host. I can't seem to figure out where to set the appropriate value in the configs.

**Comment by firef0x00** [comment]
I fixed it by adding the following lines to Dockerfile (after line 6) before building it:

```
FROM ubuntu:23.04
SHELL ["/bin/bash", "-c"]

ARG TZ="UTC"
RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
    echo $TZ > /etc/timezone

# fix the login issue with the right public ip
WORKDIR /usr/src/app/plugins/magma
RUN echo VITE_CALDERA_URL=http://<IP>:8888 > .env
```

You can also fix it after the container is running.
Open the container terminal:
`sudo docker exec -it <container> bash`

Then you can create the required file inside the container and add content:
`echo VITE_CALDERA_URL=http://<IP>:8888 > plugins/magma/.env`

Restart the container.

**Comment by ckmk14** [comment]
I encounter the same issue without docker

**Reply by ckmk14** [reply]
sorry, duplicate of https://github.com/mitre/caldera/issues/2885
